server/src/engineModel/EngineModel.ts

```diff
@@ -52,17 +52,18 @@ export class EngineModel {
     private buildYmlFunction(func: any, sourceType?: string): AbstractYmlFunction {
         let ymlAbstractFunction: AbstractYmlFunction;
         if (sourceType) {
-            ymlAbstractFunction = new YmlFunction(func.$.ident, this.uri);
+            ymlAbstractFunction = new YmlMethod(func.$.ident, this.uri);
             this.functions.push(ymlAbstractFunction);
             ymlAbstractFunction.detail = `[${sourceType}].${ymlAbstractFunction.label}`;
         } else {
-            ymlAbstractFunction = new YmlMethod(func.$.ident, this.uri);
+            ymlAbstractFunction = new YmlFunction(func.$.ident, this.uri);
             ymlAbstractFunction.detail = `[STATIC] ${ymlAbstractFunction.label}`;
         }
+        ymlAbstractFunction.setUserInformations(
+            `(${sourceType ? 'method' : 'function'}) ${ymlAbstractFunction.detail}`,
```

Comment:
* Do people care much about that function vs. method destinction?
* Is it already possible to tell whether it’s a func or meth with just what ends up in `ymlAbstractFunction.detail`?

Reply:
* I do
* This method rewrite `detail`

> * This method rewrite `detail`

Oh, that's awful. :scream: